packages/vm/tests/util.ts

```diff
@@ -144,14 +144,14 @@ export async function verifyPostConditions(state: any, testData: any, t: tape.Te
         const promise = verifyAccountPostConditions(state, address, account, testData, t)
         queue.push(promise)
       } else {
-        t.fail('invalid account in the trie: ' + <string>key)
+        t.comment('invalid account in the trie: ' + <string>key)
```

Comment:
Converted all tape commands to `comment`. The reason is that if you have failing tests, then this adds to the number of tests counter. Let's say we accidentally do not run some tests, then `--verify-amount-alltests` flag could still pass while this is actually not the case.

Also note that if any of these comments get triggered, then the state root of the block is wrong, so the test fails anyways.